remhos.cpp

```diff
@@ -600,9 +617,26 @@ int main(int argc, char *argv[])
    }
    else if (lo_type == LOSolverType::ResDist)
    {
-      const bool subcell_scheme = false;
-      lo_solver = new ResidualDistribution(pfes, k, asmbl, lumpedM,
-                                           subcell_scheme, time_dep);
+     const bool subcell_scheme = false;
+     if(pa)
```

Comment:
Styling comment -- here and below, please leave a space between `if` and `(`:

```suggestion
     if (pa)
```